boms/cloud-oss-bom/pom.xml

```diff
@@ -44,17 +44,18 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <guava.version>30.1.1-android</guava.version>
-    <google.cloud.java.version>0.158.0</google.cloud.java.version>
-    <google.cloud.core.version>1.95.4</google.cloud.core.version>
+    <guava.version>30.1.1-jre</guava.version>
+    <google.cloud.java.version>0.159.0</google.cloud.java.version>
+    <google.cloud.core.version>2.0.5</google.cloud.core.version>
```

Comment:
From `<google.core.version>2.0.5</google.core.version>` in https://search.maven.org/artifact/com.google.cloud/google-cloud-shared-dependencies/2.0.1/pom